api/api/authentication.py

```diff
@@ -94,17 +94,29 @@ def check_user(user: str, password: str, required_scopes=None) -> Union[dict, No
 _public_key_path = os.path.join(SECURITY_PATH, 'public_key.pem')
 
 
-def generate_keypair():
+def generate_keypair(curve: ec.EllipticCurve) -> Tuple[str, str]:
```

Comment:
- Since we are refactoring this function, could it be more descriptive to have it called `get_keypair` or similar? That is since it creates the files or loads their contents,
- Could we add the instantiation of the Elliptic Curve inside the function to avoid the modules that use the function to require the `from cryptography.hazmat.primitives.asymmetric import ec` line?

Reply:
1. Yes, I've updated the name to `get_keypair` as suggested.
2. The thing is that the function now accepts any kind of elliptic curve, because the APIs use different ones, so I need the function call to specify which one to create. At the end of the day these functions should be used in very few places.